src/python/pants/option/global_options.py

```diff
@@ -597,7 +596,7 @@ class BootstrapOptions:
         "--pants-workdir",
         advanced=True,
         metavar="<dir>",
-        default=os.path.join(_buildroot, ".pants.d"),
+        default=lambda cls: os.path.join(get_buildroot(), ".pants.d"),
```

Comment:
Should this be `cls._buildroot`? Or `lambda _: os.path.join(get_buildroot())`? Actually don't use the lambda at all, right - no need?

Reply:
It definitely needs to be delay-loaded (import-time vs. Pants invocation time). Not sure why other than I think the tests need to set up some prereqs before this code gets invoked. Real code is likely fine, but :man_shrugging:

Reply:
I'll definitely switch to `_` for the param. Do you think this warrants a copy+paste comment?